app/models/tag.rb

```diff
@@ -26,7 +26,7 @@ class Tag < ActsAsTaggableOn::Tag
   before_validation :evaluate_markdown
   before_validation :pound_it
   before_save :calculate_hotness_score
-  after_commit :bust_cache
+  after_commit :bust_cache, :index_to_elasticsearch
```

Comment:
+1 to keep ES in the methods, so that we can separate them from algolia in the meantime. It'd be great if we'd rename them after the switch. It's nice to think ES as just the backend one level encapsulated in the facade

Reply:
100% agree, its just a search tool and thus should be called that in code. https://github.com/thepracticaldev/dev.to/projects/6#card-32887789